frontend/src/components/ImplementationReportView/DialectCompliance.tsx

```diff
@@ -42,7 +42,18 @@ const DialectCompliance: React.FC<{
               .map(([dialectName, result], index) => {
                 return (
                   <tr key={index}>
-                    <td>{Dialect.forPath(dialectName).uri}</td>
+                    <td>
+                      {Dialect.forPath(dialectName).uri}{" "}
+                      <img
```

Comment:
Just my opinion: I feel like badges always appear as clickable links - maybe this can be a link to:

- badge image itself: https://img.shields.io/endpoint?url=https%3A%2F%2Fbowtie-json-schema.github.io%2Fbowtie%2Fbadges%2Fjava-dev.harrel.json-schema%2Fsupported_versions.json
- badge data actually hosted by bowtie: https://bowtie-json-schema.github.io/bowtie/badges/java-dev.harrel.json-schema/supported_versions.json

Reply:
Also, linter is complaining about lack of `alt` attribute - you may as well just add it

Reply:
Linking somewhere to https://shields.io/badges/endpoint-badge seems also maybe like a decent idea, to send people to how they would customize the badge if they feel like doing so.